plextraktsync/plex_server.py

```diff
@@ -58,7 +60,9 @@ def _get_plex_server():
                 # save new url to .env
                 CONFIG["PLEX_TOKEN"] = plex_token
                 CONFIG["PLEX_BASEURL"] = new_plex_baseurl
-                CONFIG["PLEX_FALLBACKURL"] = plex_fallbackurl
+                plex_localurl = CONFIG.get("PLEX_LOCALURL")
```

Comment:
have you tested does `CONFIG.get("PLEX_LOCALURL")` thow if key is missing? maybe need `CONFIG.get("PLEX_LOCALURL", None)`

Reply:
yes i tested this